packages/nimble-components/src/table/index.ts

```diff
@@ -126,22 +137,31 @@ export class Table<
     private readonly tableValidator = new TableValidator();
     private readonly updateTracker = new UpdateTracker(this);
     private columnNotifiers: Notifier[] = [];
+    private readonly collapsedRows = new Set<string>();
 
     public constructor() {
         super();
         this.options = {
             data: [],
             onStateChange: (_: TanStackUpdater<TanStackTableState>) => {},
+            onExpandedChange: this.handleExpandedChange,
             getCoreRowModel: tanStackGetCoreRowModel(),
             getSortedRowModel: tanStackGetSortedRowModel(),
+            getGroupedRowModel: tanStackGetGroupedRowModel(),
+            getExpandedRowModel: tanStackGetExpandedRowModel(),
+            getIsRowExpanded: this.getIsRowExpanded,
             columns: [],
-            state: {},
+            state: {
+                grouping: [],
+                expanded: true
```

Comment:
I've you've created that issue to TanStack yet, it's probably worth referencing it from here. Having to set `expanded: true` is a workaround for that issue given your custom implementation of `getIsRowExpanded`